tests/mail/tests.py

```diff
@@ -1037,9 +1049,14 @@ def test_dont_base64_encode(self):
         )
         s = msg.message().as_bytes()
         self.assertIn(b"Content-Transfer-Encoding: 8bit", s)
+        # TODO: modern-email: remove next assertion.
+        #   8bit CTE within a Unicode str is not meaningful, and modern email
+        #   won't generate it. (This is left over from Python 2.)
```

Comment:
```suggestion
```
We should remove the TODOs as we can merge commits separately and don't add in references to future work

Reply:
That's OK with me.

The tradeoff is that the diff in the future work is harder to understand: it either looks like a test might have been accidentally removed, or the future work has to leave behind a tombstone comment explaining the change ([example above](https://github.com/django/django/pull/18502#discussion_r1799776083)).

An alternative could be to add a (non-TODO) comment explaining why this test is here now, and then remove both that comment and the test together in the future work. E.g.:

```py
        s = msg.message().as_bytes()
        self.assertIn(b"Content-Transfer-Encoding: 8bit", s)
        # The following test is left over from Python 2 and can be safely removed.
        # 8bit CTE within a Unicode str is not meaningful, and Python's modern 
        # email api won't generate it. (The test still works with the legacy api.)
        s = msg.message().as_string()
        self.assertIn("Content-Transfer-Encoding: 8bit", s)
```

Reply:
I like the non-todo comment :+1: sounds good
Appreciate the thought into the reviewing :people_hugging: